tests/UnifiedSpecTests/CollectionData.php

```diff
@@ -3,7 +3,6 @@
 namespace MongoDB\Tests\UnifiedSpecTests;
 
 use ArrayIterator;
-use IteratorIterator;
```

Comment:
Does this PR basically remove all references to IteratorIterator in the library's test suite -- or at least any place we used it to wrap a Cursor?

Reply:
Yes, we did this in a number of tests to be able to work with the cursor directly (e.g. by using a `MultipleIterator`, which requires an `Iterator`).

Other places where we wrap a cursor are the various `*InfoIterators` which remain as they add functionality, and `ChangeStreamIterator`, which also attaches special handling and thus can't not wrap the original cursor.